io.bit3.WhatsAppQT.yml

```diff
@@ -0,0 +1,34 @@
+app-id: io.bit3.WhatsAppQT
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+base: io.qt.qtwebengine.BaseApp
+base-version: '5.14'
+command: WhatsAppQT
+finish-args:
+- "--share=ipc"
+- "--socket=x11"
+- "--socket=wayland"
+- "--share=network"
+- "--filesystem=home"
```

Comment:
Does it really need full home access?

Reply:
I almost said it was personal preference. I don't like it when I want to send files and have to move them to Downloads first. And vice versa, if I want to download files, I want to save them directly to a certain location and not in Downloads to move them from there.

But I can switch that to "xdg-downloads" if that's preferred. I can still overwrite the access locally.